src/ML-PACE/pair_pace_al.cpp

```diff
@@ -0,0 +1,599 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/*
+Copyright 2022 Yury Lysogorskiy^1, Anton Bochkarev^1, Matous Mrovec^1, Ralf Drautz^1
+
+^1: Ruhr-University Bochum, Bochum, Germany
+ */
+
+
+//
+// Created by Lysogorskiy Yury on 2.01.22.
+//
+
+#include <cmath>
```

Comment:
Please see https://docs.lammps.org/Modify_style.html#programming-conventions-varied
for our conventions about which files to include and their order (and other programming conventions).